supersuit/generic_wrappers/no_op_reset.py

```diff
@@ -0,0 +1,75 @@
+from supersuit.utils.frame_skip import check_transform_frameskip
+from supersuit.utils.wrapper_chooser import WrapperChooser
+from pettingzoo.utils.wrappers import BaseWrapper, BaseParallelWraper
+import gym
+from supersuit.utils.make_defaultdict import make_defaultdict
+import numpy as np
+    
+class noop_reset_gym(gym.Wrapper):
+    def __init__(self, env, noop_max=30):
+        super().__init__(env)
+        self.noop_max = noop_max
+        self.override_num_noops = None
+        self.noop_action = 0
+
+    def reset(self, **kwargs):
+        obs = self.env.reset(**kwargs)
+        if self.override_num_noops is not None:
+            noops = self.override_num_noops
+        else:
+            noops = self.unwrapped.np_random.randint(1, self.noop_max + 1)
+        assert noops > 0
+
+        for _ in range(noops):
+            obs, _, done, _ = self.env.step(self.noop_action)
+            if done:
+                obs = self.env.reset(**kwargs)
+        return obs
+
+class noop_reset_aec(BaseWrapper):
```

Comment:
If you remove this, it should still work, as the WrapperChooser will automatically be able to convert to and from AEC if the environment `is_parallelizable` noop_reset_par. AEC is super unintuitive for these sorts of issues. So I think this should be removed.